### PR TITLE
Replace deprecated code

### PR DESCRIPTION
**Pull request by aribronstein**
Replace React.createClass with ES class 
Replace React.PropTypes with standalone npm library prop-types

#32 
#33

### CODE REVIEW

File: src/Component.js [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
 
 const MAX_MOVE = 20;
 
 
 const extractCoordinates = ({changedTouches}) =>
   ({x: changedTouches[0].screenX, y: changedTouches[0].screenY});
 
-
-export const ReactPageClick = React.createClass({
+class ReactPageClick extends React.Component {
```
Comment (nkbt) [inline src/Component.js]: Please, also switch to `React.PureComponent` so we can remove `import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';` line and shouldComponentUpdate too

File: package.json [hunk 2]
```diff
@@ -40,6 +40,7 @@
     "react": "^0.14 || ^15"
```
Comment (nkbt) [inline package.json]: oh, almost forgot. Pure component requires react@15.3, so `peerDependencies` should be updated to have `"react": ">=15.3"`.

File: package.json [hunk 3]
```diff
@@ -37,9 +37,10 @@
   },
   "homepage": "https://github.com/nkbt/react-page-click",
   "peerDependencies": {
-    "react": "^0.14 || ^15"
+    "react": ">=15.3"
   },
   "devDependencies": {
+    "prop-types": "^15.5.8",
```
Comment (nkbt) [inline package.json]: Wanted to merge, but just noticed that `prop-types` is in `devDependencies`. It needs to be in `dependencies`

Sorry for missing it right away

Comment (aribronstein) [reply package.json]: dependencies or peerDependencies?

Comment (nkbt) [reply package.json]: dependencies

**Comment by nkbt** [comment]
Great work @aribronstein! Thanks

I will publish new version tomorrow

**Pull request merged**